uint8 robot
uint8 enemy
---
bool success